### PR TITLE
Milestone 4: refactor

### PR DESCRIPTION
**Pull request by ookino**
Refactored Calculator.js from a class-based component to a functional component

### CODE REVIEW

File: src/components/calculator/Calculator.js [hunk 1]
```diff
@@ -1,163 +1,158 @@
...
-          onClick={this.handleClick}
-        />
-        <input
-          value="0"
-          type="button"
-          className="calculator__key calculator__key--zero calculator__padding"
-          onClick={this.handleClick}
-        />
+      <input
+        className="calculator__key calculator__key--operator calculator__padding"
```
Comment (Davidosky007) [inline src/components/calculator/Calculator.js:129]: * I cant perform division operation kindly fix it 🙏 (ie 30 divided by 10)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @yaseerokino 👋, 

Good job so far! 💪 
* Nice looking readme file ✔️ 
* Well written PR ✔️ 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/calculator/Calculator.js (hunk 1 above)
Comment (ookino) [reply src/components/calculator/Calculator.js:129]: @ Davidosky007 i have made the required changes

**Review by AshakaE — APPROVED** [review]
### Hi @yaseerokino  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested change 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ookino** [comment]
@AshakaE thank you for the review, I thought I was already using the [optional] prefix. could you give me an example of how I should use the optional prefix

**Pull request merged**